docs/jumpstart/week-7.md

```diff
@@ -0,0 +1,512 @@
+assigned to an anonymous asynchronous function. The function is then going to make a get request
+using our `AxiosInstance` we created previously. `.get()` requires one argument, which is the url
+to make the get request at. As you recalled, the `AxiosInstance` already has our baseURL to our
+backend so all we need is to go to that url plus "/posts", which we can use the constant we 
+described. We follow up all of our get requests with `.then()` in order to break down the response
+into the data we are requesting. To do this we make an anonymous function that takes in a response
+(what the promise returned) and then dissects the response to what we want.
+
+Here is another get request example:
+
+```
+export const getPost = async (id: number): Promise<BlogPost> => AxiosInstance.get(`${POSTS}/${id}`).then((response) => response.data);
+```
+
+In this call we need to take in the specific post number as an argument to the anonymous function
+so that we can use it in the `get()` call in the function body.
+
+> Notice that we are giving types to the arguments of the anonymous functions, the return type of 
+>the anonymous function and the argument portion of the anonymous function.
+
+
+##### Post Request
+
+Now, what can we use a post request for? That is right! We can use it to create a post/comment and 
+clap a post/comment. Let's take a look at how to make a post request:
+
+```
+export const createPost = async (post: CreatePost): Promise<void> => AxiosInstance.post(`${POSTS}`, post);
+export const clapPost = async (id: number): Promise<void> => AxiosInstance.post(`${POSTS}/${id}/clap`);
+```
+
+Just like before we have an anonymous function that has an `AxiosInstance` to make the request. The
+difference is that this time the function takes in an argument. This argument is going to be the
+data that we want to post to the backend. Now, to make a post request, `post()` takes in the url 
+location of where to post the data, and the data to post at that location in the form 
+`post(location,data)`. Let's look specifically at `createPost`. We know that we want to "post" the 
+blog post to the backend. So, we take in a post as an argument. In the `post()` call we give the 
+url (which in this case is `"/posts"`), and the data we want to post (the argument we took in the 
+anonymous function). 
+
+That's all there is to it.   
+
+##### Delete Request
+
+The delete request will be the last type of request that we will use. What do you think we can use
+it for? Right again! We can use the delete request to delete a post/comment. Let's take a look at
+how to do that:
+
+```
+export const deletePost = async (id: number) => AxiosInstance.delete(`${POSTS}/${id}`);
+``` 
+
+Same as before. We have an anonymous function that takes in the post number to delete. We then use
+our `AxiosInstance` to call delete the id from the specific url. It is simliar to our get request
+excpet we are not getting but deleting.
```

Comment:
```suggestion
our `AxiosInstance` to make a call to delete the id from the specific url. It is simliar to our `GET` request
except we are not deleting instead of getting.
```